indico/modules/events/editing/client/js/management/editable_type/EditableList.jsx

```diff
@@ -325,8 +325,8 @@ function EditableListDisplay({
         searchableId: c.friendlyId,
         searchableFields:
           c.editable && c.editable.editor
-            ? [c.title, c.code, c.editable.editor.fullName]
-            : [c.title, c.code],
+            ? [c.title, c.code, c.persons.join(''), c.editable.editor.fullName]
+            : [c.title, c.code, c.persons.join('')],
```

Comment:
```suggestion
            ? [c.title, c.code, c.persons.join('|'), c.editable.editor.fullName]
            : [c.title, c.code, c.persons.join('|')],
```

Otherwise you could get some false positives from concatenated names

Reply:
or maybe a character that can't be typed in the field, like `\n`? seems to work fine

Reply:
with a typical large event (that has many editables *and* many authors on each of them)? if yes fine to leave

Reply:
yep I'm doing all tests with the IPAC23 data